### PR TITLE
Sync configs from file 121 in the `bootstrap.properties` and `application.properties`

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Cherry-pick https://github.com/hashgraph/hedera-services/pull/9055

### CODE REVIEW

**Review by ElijahLynn — DISMISSED** [review]
Thanks!

**Review by Nana-EC — APPROVED** [review]

File: hedera-node/configuration/mainnet/bootstrap.properties [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+staking.fees.stakingRewardPercentage=10
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+tokens.nfts.maxAllowedMints=10_000_000
+accounts.maxNumber=20_000_000
+#Overrides that differ based on the network
+ledger.id=0x01
+contracts.chainId=296
```
Comment (rbair23) [inline hedera-node/configuration/mainnet/bootstrap.properties:21]: Wrong values

Comment (rbair23) [reply hedera-node/configuration/mainnet/bootstrap.properties:21]: Or at least, inconsistent with application.properties

Comment (Neeharika-Sompalli) [reply hedera-node/configuration/mainnet/bootstrap.properties]: @rbair23  Yes, thank you! Updated

Comment (Neeharika-Sompalli) [reply hedera-node/configuration/mainnet/bootstrap.properties]: Updating develop with this PR https://github.com/hashgraph/hedera-services/pull/9079

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**